### ISSUE TITLE
Exodus - Ability to create a My List

### ISSUE THREAD

**Issue by eyezonly** [lambda81/lambda-addons#87]
Is it possible to add a My List option fo Exodus? Similar to that of Netflix, when I'm browsing shows or tv series, I find some that interest me & wish to watch them at a later time. Instead of having to write down the movie or tv series as a reminder, a simple add to My List option would allow me to keep a list of future movies or shows to watch at a later time. Keep up the good work! Great add-on!

**Comment by 1BobDole** [comment]
Yes. I would like to second that request. Right now Exodus only allows use of trakt, which openly states that they make your history public, and will turn over all information to attorneys and law enforcement with a simple request, even without a warrant. That is an unacceptable service for a Kodi application, as it completely violates privacy. A simple favourites option, like the one on Genesis, would be perfectly fine.

**Comment by MehBlorg** [comment]
So what? They just track what you watched, not where you watched it from. Or do they? Can you provide a link to their statement about this please?

**Comment by 1BobDole** [comment]
Yes, they openly stated they will turn over ALL files. This includes who was watching what, all times and sources. Here is a link to one of the articles, and you can also find these statements in trakt's own "terms and conditions" statements: [https://torrentfreak.com/pirates-expose-their-download-habits-through-trakt-141130/](url)
